stubs/Symfony/Component/Form/FormInterface.stub

```diff
@@ -3,10 +3,15 @@
 namespace Symfony\Component\Form;
 
 /**
- * @extends \ArrayAccess<string, \Symfony\Component\Form\FormInterface>
- * @extends \Traversable<string, \Symfony\Component\Form\FormInterface>
+ * @template TData
+ *
+ * @extends \ArrayAccess<string, \Symfony\Component\Form\FormInterface<mixed>>
+ * @extends \Traversable<string, \Symfony\Component\Form\FormInterface<mixed>>
  */
 interface FormInterface extends \ArrayAccess, \Traversable, \Countable
 {
-
+    /**
+     * @return TData
```

Comment:
I didn't look where is the truth, but the Psalm template returns `?TData`
https://github.com/psalm/psalm-plugin-symfony/blob/5.x/src/Stubs/common/Component/Form/FormInterface.stubphp#L12

Reply:
I think it will be null only if we omit a initial value of the form.
https://github.com/siketyan/phpstan-symfony/blob/88da905c8a29f791fa9ee78ec5f83f2980f8e809/stubs/Symfony/Component/Form/FormFactoryInterface.stub#L17

Reply:
I see. So it means psalm should improve his typing ^^